package.json

```diff
@@ -63,7 +63,7 @@
     "passport-local": "^1.0.0"
   },
   "devDependencies": {
-    "async": "^2.1.5",
+    "async": "^2.6.3",
```

Comment:
Any particular reason for updating this? just wondering.

Reply:
Nope, I think it was automatically updated by the package manager. I just changed it back.